amundsen_application/static/js/components/common/ResourceListItem/TableListItem/index.tsx

```diff
@@ -3,10 +3,8 @@ import { Link } from 'react-router-dom';
 
 import { LoggingParams, TableResource} from '../types';
 
-import './styles.scss';
-
 interface TableListItemProps {
-  item: TableResource;
+  table: TableResource;
   logging: LoggingParams;
```

Comment:
Slipped my mind previously -- `logging` should be optional on all the list item components. We use these values for instrumentation when a user clicks on the item from the search results, but we won't be using them when someone clicks on the item from the profile page.

Reply:
As I was revisiting the logging parameters, I realized there are deep changes that I would like to make. I think it makes sense to break this into a separate PR as not to pollute this one.

Reply:
Of the logging parameters I think `source` should be required, but `index` should be optional. The SearchList has optional logging params, but I think they should be required. Etc.

Reply:
I'm not sure I understand the thinking, can you elaborate? I *think* I understand the idea behind keeping `source` -- we could send something like `profile_page` as the `source` for other options and not send the `index` because it doesn't matter in that case. Is that where you're coming from?

Reply:
Yes, every time you display a `ResoureListItem` it will have a `source` that should be logged. I suppose they will also have an `index` given that they are in a list. Whether or not recording that index is useful is another question of which I am doubtful.

If anything this would be an argument for making the logging params mandatory for all cases, rather than making them optional.

Reply:
^ alright lets complete this discussion when that PR comes.